### PR TITLE
test: [RLOS2023] [WIP] add more arguments for reg&cb tests

### PR DESCRIPTION
**Pull request by michiboo**

### CODE REVIEW

File: python/tests/test_regs.json [hunk 1]
```diff
@@ -37,21 +37,54 @@
...
+            {
+                "--decay_learning_rate": [
+                    1.1,
+                    1,
+                    0.9
+                ]
+            },
+            {
+                "--power_t": [
+                    0.5,
```
Comment (ataymano) [inline python/tests/test_regs.json:65]: Can you please add nullls to test default values aas well

File: python/tests/test_regs.json [hunk 2]
```diff
@@ -37,21 +37,54 @@
                     "-P 50000 --preserve_performance_counters --save_resume "
                 ]
             },
+            "(",
```
Comment (ataymano) [inline python/tests/test_regs.json:40]: Can you please try different loss functions as well?
https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Loss-functions

File: python/tests/test_regs.json [hunk 3]
```diff
@@ -37,21 +37,54 @@
...
             {
                 "#reg": [
                     "--coin",
-                    "--ftrl",
```
Comment (ataymano) [inline python/tests/test_regs.json:43]: [nit] other reductions that may be interesting to add:
--freegrad
--bfgs
--conjugate_gradient

File: python/tests/test_cb.json [hunk 4]
```diff
@@ -157,7 +188,39 @@
...
+            },
+            "(",
+            {
+                "--epsilon": [
+                    0.1,
+                    0.2,
+                    0.3
+                ]
+            },
+            "+",
```
Comment (ataymano) [inline python/tests/test_cb.json:200]: other exploration algos (--cb_explore_adf_***):
https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/vowpalwabbit/core/src/reductions/cb
Improtant stuff to add:
--squarecb
--synthcover
--regcb
--softmax

File: python/tests/test_cb.json [hunk 5]
```diff
@@ -157,7 +188,39 @@
                 "#base": [
                     "--cb_explore_adf"
                 ]
-            }
+            },
+            "(",
+            {
```
Comment (ataymano) [inline python/tests/test_cb.json:193]: consider adding base learner options to the grid
